bin/install

```diff
@@ -186,7 +186,7 @@ download_and_verify_checksums() {
           export GNUPGHOME="${ASDF_DIR:-$HOME/.asdf}/keyrings/nodejs"
       fi
 
-      if ! $gnugp_verify_command_name --display-charset utf-8 --verify "$signed_checksum_file"; then
+      if ! $gnugp_verify_command_name --display-charset utf-8 --verify "$signed_checksum_file" 2>/dev/null; then
```

Comment:
Warnings from this command confuse users, and I don't think they are relevant right now.